src/index.html

```diff
@@ -11,6 +11,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- big search bar -->
+    <form
+      action="#"
+      class="bigSearchBar"
+      method="get"
+    >
+      <label class="searchBar">
+        <img 
+          class="image"
+          src="images/Search.svg" 
+          alt="a" 
+        >
+        <input 
+          type="text"
+          class="firstSearchBar "
+          data-qa="big"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <!-- big search bar -->
+    
+    
+    <!-- small search bar -->
+    <form
+      action="#"
+      class="smallSearchBar"
+      method="get"
+    >
+      <label class="searchBar searchBar--small">
+        <img 
+          src="images/Search.svg" 
+          alt="a" 
+          class="image image--small"
+        >
+        <input 
+          type="text"
+          class="firstSearchBar firstSearchBar--small"
```

Comment:
you don't need modifier `small` for `image/firstSearchBar`, bc you can style `.searchBar--small .image/.firstSearchBar`